zap/src/output/luau/client.rs

```diff
@@ -158,10 +139,29 @@ impl<'src> ClientOutput<'src> {
 		self.dedent();
 		self.push_line("end");
 		self.dedent();
+	}
 
-		if self.config.manual_event_loop {
-			self.push_line("end");
-		} else {
+	fn push_event_loop(&mut self) {
+		self.push("\n");
+
+		let send_events = self.config.casing.with("SendEvents", "sendEvents", "send_events");
+
+		self.push_line(&format!("local function {send_events}()"));
+		self.indent();
+		self.push_event_loop_body();
+		self.push_line("end\n");
+
+		if !self.config.manual_event_loop {
+			self.push_line("local time = 0\n");
+			self.push_line("RunService.Heartbeat:Connect(function(dt)");
+			self.indent();
+			self.push_line("time += dt");
+			self.push("\n");
+			self.push_line("if time >= (1 / 61) then");
+			self.indent();
+			self.push_line("time -= (1 / 61)");
```

Comment:
Let's get rid of this and the `time` variable now that roblox supports up to 240 FPS on windows.